src/style.css

```diff
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #e5e5f7;
+  opacity: 0.8;
+  background-size: 20px 20px;
+  background-image: repeating-linear-gradient(0deg, #444cf7, #444cf7 1px, #e5e5f7 1px, #e5e5f7);
+}
+
+.container {
+  display: flex;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 4px solid yellow;
+  padding: 2rem;
+  background: rgba(0, 0, 0, 0.808);
+  color: white;
+}
+
+.list {
+  padding: 2rem;
+}
```

Comment:
- [ ] Well done, I can see you put in a lot to make this app work. There is just one thing remaining to scale through this milestone. It would be nice to make the styling of your app match that of the wireframe used in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). You don't have to include all features yet, but the design should match that of the wireframe. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/160041333-f5caae7e-42b4-4632-aacc-6f71677485b8.png)


### Wireframe

![image](https://user-images.githubusercontent.com/77106962/160041407-97ed68dc-e2f1-4c62-b7d7-0c657aa05635.png)

#### Kindly implement it 👍